### PR TITLE
HA test for stolon on k8s 

### PR DESCRIPTION
**Pull request by agalitsyn**

### CODE REVIEW

File: images/stolonctl/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.4
```
Comment (sofuture) [inline images/stolonctl/Dockerfile]: Let's use our debian images, they come with curl for example.

Comment (agalitsyn) [inline images/stolonctl/Dockerfile]: :+1: Alexey told this to me today

**Comment by sofuture** [comment]
@agalitsyn and I discussed earlier today in slack. Had a stack explosion trying to run the test.

**Comment by agalitsyn** [comment]
@sofuture please review new commits

**Comment by sofuture** [comment]
lgtm -- I had a problem with some of the changed labels, but fixed in e4733bc

**Comment by agalitsyn** [comment]
I missed it, thanks!

**Pull request merged**